backend/addsclient.js

```diff
@@ -3,48 +3,79 @@ const xml2js = require('xml2js');
 
 // Client for the NWS ADDS service
 const AddsClient = {
-  baseUrl: 'https://aviationweather.gov/adds/dataserver_current/httpparam?requestType=retrieve&format=xml&',
```

Comment:
Looks like the bulk of this file's changes are whitespace.  Is there a preferred indent/line ending standard?

Reply:
My original JS linter broke, so I went to a new one. The new standard is 4 spaces, however most files don't reflect that